presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -19,6 +19,31 @@ plugin and enabled by adding ``resource-groups.configuration-manager=file`` to
 ``etc/resource-groups.properties`` and setting ``resource-groups.config-file`` to the
 location of a JSON config file with the properties described below.
 
+To use an implementation that is backed by a database ``resource-groups.configuration-manager=db``
```

Comment:
This is added in the middle of explaining file based resource group configuration. Could you please move this to after the json based example? Or maybe mention available configuration managers here (file, db) and explain each in different sections.